### PR TITLE
Release v0.42.0

### PR DESCRIPTION
**Pull request by Nexucis**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
...
+- [BUGFIX] Remove extra call to `setDatasourceDrawerOpened` in DatasourceList (#1597)
+- [BUGFIX] Fix typo in QuerySummaryTable (#1596)
+- [BUGFIX] Fix YAML files not working for provisioning (#1589)
+- [BUGFIX] ListVariable: fix scroll reset after a new option is selected (#1579)
+- [BUGFIX] Saved value for allowMultiple was not retrieved (#1577)
+- [BUGFIX] Fix several issues with JSON views (#1562)
+- [BUGFIX] Avoid failing grafana migration in multiple cases (#1549)
+- [BUGFIX] Fix unwanted thresholds when migrating timeseries panels (#1495)
+- [BUGFIX] Fix grafana migration failing in some corner cases (#1487)
+- [BUGFIX] Fix too-restrictive regex used for URL validation (#1470)
```
Comment (AntoineThebaud) [inline CHANGELOG.md:42]: this one was already part of the previous release https://github.com/perses/perses/releases/tag/v0.41.1

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+- [FEATURE] Adding explore page (#1477) (#1592) (#1610)
+- [FEATURE] Enabling data provisioning (#1486)
+- [FEATURE] Add sorting capability for ListVariable (#1502)
+- [FEATURE] Add Secret & GlobalSecret list view (#1501)
+- [FEATURE] Display the server config (#1496)
+- [ENHANCEMENT] Prevent using dashboard datasources before they're saved (#1603)
+- [ENHANCEMENT] add a default encryption key (#1599)
+- [ENHANCEMENT] Use a context for the config to share it across the whole app (#1604)
+- [ENHANCEMENT] Implement the `healthcheck` method for Prometheus datasources (#1595)
+- [ENHANCEMENT] Local datasources CRUD (#1587)
```
Comment (AntoineThebaud) [inline CHANGELOG.md:17]: This one is more a feature actually - my bad on the wrong flagging here. Could you fix it in the changelog?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## 0.42.0 / 2023-12-07
+
+- [FEATURE] Initial Tempo Plugin (#1561)
+- [FEATURE] Add authentication system (#1472) (#1473) (#1476) (#1478) (#1484) (#1547) (#1555) (#1557) (#1591) (#1602) (#1612) (#1615)
+- [FEATURE] Add authorization system based on rbac (#1570) (#1511) (#1613) (#1601) (#1605) (#1626)
```
Comment (Gladorme) [inline CHANGELOG.md:7]:
```suggestion
- [FEATURE] Add authorization system based on rbac (#1570) (#1511) (#1613) (#1601) (#1605) (#1626) (#1627]
```
Soon 😋

**Review by Gladorme — APPROVED** [review]
LGTM 🚀

**Review by AntoineThebaud — APPROVED** [review]

**Pull request merged**